TriblerGUI/widgets/downloadprogressbar.py

```diff
@@ -1,4 +1,7 @@
+from __future__ import absolute_import
+
 import base64
+from six.moves import xrange
```

Comment:
Please put this after the `PyQt5` and move the `import math` after the `import base64`.